### ISSUE TITLE
Uncheck errors inspections

### ISSUE THREAD

**Issue by zolotov** [go-lang-plugin-org/go-lang-idea-plugin#2081]
Related to https://github.com/kisielk/errcheck

1. Add assignment error to blank identifier inspection
2. Add ignored error inspection

**Comment by dlsniper** [comment]
Not sure what you mean about 2., the ignored error part, but for 1. I think we already have the issue in place: https://github.com/go-lang-plugin-org/go-lang-idea-plugin/issues/1692

**Comment by zolotov** [comment]
No, #1692 is about 2. Follow the link to know what I mean about assignment to blank inspection
